src/02_evaluate_better_models.R

```diff
@@ -2,17 +2,17 @@ source("src/dimorphism_model_class.R")
 source("src/calculator_ROC_class.R")
 
 directorioTDP <- ("data/raw")
-nombreArchivoCSV <- file.path(directorioTDP,"morfometria_albatros-laysan_guadalupe.csv")
+nombreArchivoCSV <- file.path(directorioTDP, "morfometria_albatros-laysan_guadalupe.csv")
 
 Datos <- data.table::data.table(read.csv(nombreArchivoCSV))
 
 ruta_resultados <- "data/processed/"
-tabla_importada <- data.table::data.table(readr::read_csv(paste0(ruta_resultados,"tabla_modelos_logisticos.csv")))
+tabla_importada <- data.table::data.table(readr::read_csv(paste0(ruta_resultados, "tabla_modelos_logisticos.csv")))
 calculadorROC <- ROC$new()
 n_renglones <- nrow(tabla_importada)
 errores <- c()
 
-for(i_renglon in 1:n_renglones) {
+ for (i_renglon in 1:n_renglones) {
```

Comment:
Creo que te sobra un espacio al inicio:
```suggestion
for (i_renglon in 1:n_renglones) {
```